NightscoutUploadKit/DeviceStatus/BatteryStatus.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BatteryStatus.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 7/28/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public class BatteryStatus {
```

Comment:
Just that I haven't been indoctrinated fully.  Getting closer, thanks to you. :)